project/src/main/java/com/google/sps/data/Places.java

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * The package that currently holds all the java files of the food-mood project.
+ */
+package com.google.sps.data;
+
+import com.google.appengine.repackaged.com.google.common.collect.ImmutableList;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Collections;
+
+/**
+ * A utility class for Place objects.
+ */
+public final class Places {
```

Comment:
Notice this class missing a unit test as well.

More of an FYI (and in case you noticed ignore my comment), but the reason that it's ok not to write unit tests for this method is that as this involves a random output (and you delegate the randomness to outer implementation), you can't know what the end result will be. However, if one so desires, you can validate that the output contains all elements as the input, or the sizes, etc. (IMO it's ok not to write one, but note that once you add another sort method to Places, a unit test is a must).

Reply:
that's a great point. will add tests when we implement more sorting methods. thank you